README.md

```diff
@@ -77,15 +77,15 @@ Open the console in your chrome debugger to see the data that is returned from t
 
 - Now lets write the response as an unordered list into the Suggestion span with id #txtHint.
 
-```
-  var everything;
-  everything = "<ul>";
-  for (let i=0; i < json.length; i++) {
-    everything += "<li> "+json[i].city;
-  };
-    
-  everything += "</ul>";
-  document.getElementById("txtHint").innerHTML=everything;
+```js
+var everything;
```

Comment:
This example is _very_ prone to code injection, and assumes that the source data is free of errors or malicious data. For example, if one of the `city` values has some HTML in there, like `</li> <input type="text" /> <li>` for example, we no longer control our own markup. This seems bad practice to me, and would need to be "un-taught" later.

The example works identically if we instead create the DOM nodes dynamically, and wrap remote content in a text node, like so:
```js
let everything = document.createElement("ul");
for (let i = 0; i < json.length; i++) {
  let item = document.createElement("li");
  item.appendChild(document.createTextNode(json[i].city));
  everything.appendChild(item);
}

document.getElementById("txtHint").appendChild(everything);
```

This PR was mainly about making the formatting nicer, but I'd gladly splice this change in throughout the rest of the tutorial if y'all would like.

Reply:
I love that you are considering security from the start. Frankly the whole class needs to be reworked so that security is taught from the beginning. In this specific case the city text is coming from the endpoint and so I'm not too concerned about the possibility for injection, and I worry about the slight increase in complexity while trying to learn other concepts. With that said, I think your point is valid. If you want to do the work then I will merge it. Thanks!